src/NuGet.Core/NuGet.ProjectModel/ProjectRestoreMetadata.cs

```diff
@@ -50,6 +50,9 @@ public class ProjectRestoreMetadata : IEquatable<ProjectRestoreMetadata>
         /// </summary>
         public string PackagesPath { get; set; }
 
+        public string CacheFilePath {
+            get { return OutputPath + "project.nuget.json"; }  }
```

Comment:
I don't believe the obj folder is read yet for project.json, you will need to add this to VS and msbuild. You may want to leave this feature off for project.json until everything else is done.